src/Transfer/EzPlatform/Data/LanguageObject.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of Transfer.
+ *
+ * For the full copyright and license information, please view the LICENSE file located
+ * in the root directory.
+ */
+
+namespace Transfer\EzPlatform\Data;
+
+/**
+ * Content type object.
+ */
+class LanguageObject
+{
+    /**
+     * @var string
+     */
+    public $code;
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * LanguageObject constructor.
+     *
+     * @param string $code
+     */
+    public function __construct($code)
+    {
+        $this->code = $code;
+        $this->setName();
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * @param string $name
+     *
+     * @throws \Exception
+     */
+    protected function setName($name = '')
+    {
+        if (empty($name)) {
+            if (isset($this->defaultNameMapping[$this->code])) {
+                $name = $this->defaultNameMapping[$this->code];
+            } else {
+                throw new \Exception(sprintf('Default languagename for code "%s" not found.', $this->code));
+            }
+        }
+        $this->name = $name;
+    }
+
+    /**
+     * @var array
+     */
+    protected $defaultNameMapping = array(
```

Comment:
eZSystems confirmed this on Slack (https://ezcommunity.slack.com/archives/ezplatform-use/p1451227067001211). I don't see any other way for now than to have them predefined.